teachertool/src/config.json

```diff
@@ -0,0 +1,3 @@
+{
+    "SoundEffectsVolume": 0.5
```

Comment:
It's ok to keep this file in the project, but want to avoid putting debug-only settings here. Let's find a better way to do that through the environment.